lib/index.js

```diff
@@ -70,7 +70,8 @@ export Shortcut from './Shortcut';
 export ShortcutIcon from './Shortcut/ShortcutIcon';
 export ShortcutTrigger from './ShortcutTrigger';
 export SectionHeader from './SectionHeader';
-export Tabs from './Tabs';
+export TabsLegacy from './Tabs';
```

Comment:
💚 Can we change this filename to be `./TabsLegacy`, that way the new tab can just be called `./Tabs`?